src/Reporters/Reporter.cpp

```diff
@@ -7,15 +7,12 @@
 
 #include "Reporter.h"
 #include "ConsoleReporter.h"
-#include "GuiReporter.h"
```

Comment:
we should keep GUI reporter  as a template code for future use, e.i. reuse  JAVA GUI in future.